content/v2.0/write-data/no-code/scrape-data/_index.md

```diff
@@ -8,6 +8,7 @@ description: >
 aliases:
   - /v2.0/collect-data/scraper-metrics-endpoint
   - /v2.0/collect-data/scrape-data
+    - /v2.0/write-data/scrape-data
```

Comment:
The indentation on this needs to be the same as the others. YAML is very strict with indentation. I'm a bit surprised this didn't cause an error in the build.

Reply:
interesting it didn't - thanks, Scott!